Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -703,13 +728,13 @@ void SkylanderPortalWindow::RefreshList()
     auto& system = Core::System::GetInstance();
     for (const auto& file : collection.entryInfoList(QStringList(QStringLiteral("*.sky"))))
     {
-      File::IOFile sky_file(file.filePath().toStdString(), "r+b");
-      if (!sky_file)
+      File::IOFile m_sky_file(file.filePath().toStdString(), "r+b");
+      if (!m_sky_file)
```

Comment:
m_sky_file is a member of SkylanderFigure, not SkylanderPortalWindow. This is just a local variable so we don't need the m_ prefix, and likewise for other instances later.